liquibase-core/src/main/java/liquibase/snapshot/jvm/ForeignKeySnapshotGenerator.java

```diff
@@ -4,13 +4,11 @@
 import liquibase.database.AbstractJdbcDatabase;
 import liquibase.database.Database;
 import liquibase.database.DatabaseConnection;
-import liquibase.database.core.DB2Database;
 import liquibase.database.core.Db2zDatabase;
 import liquibase.database.core.MSSQLDatabase;
 import liquibase.database.core.OracleDatabase;
+import liquibase.database.core.SybaseDatabase;
 import liquibase.database.jvm.JdbcConnection;
-import liquibase.database.core.MariaDBDatabase;
```

Comment:
Were these imports for DB2Database, MSSQLDatabase and MariaDBDatabase removed by accident?

Reply:
No accidentally. With the merge from master, I did a "fix code layout" which also removes unneeded imports. If they were needed, the compile would have failed.